### PR TITLE
Cart page

### PR DESCRIPTION
**Pull request by DivyanshSareen**
Preview: https://deploy-preview-3--pandapanda.netlify.app/cart.html
image

### CODE REVIEW

File: cart.html [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+                    </div>
+                    <div class="card-option">
+                        <button class="btn">Remove From Cart</button>
+                        <button class="btn">Move to Wishlist</button>
+                    </div>
+                </div>
+            </div>
+            <div class="card card-horizontal">
+                <div class="card-img">
+                    <img src="https://images.pexels.com/photos/3927227/pexels-photo-3927227.jpeg?cs=srgb&dl=pexels-cottonbro-3927227.jpg&fm=jpg"
```
Comment (dev-prabhat) [inline cart.html]: you can download this photo in your project

**Review by sanjitsarkar — APPROVED** [review]
Looks good

**Review by dev-prabhat — CHANGES_REQUESTED** [review]
UI looks clean color combine is good, try to avoid using multiple div tag

**Review by dev-prabhat — APPROVED** [review]
Yeap good to merge

**Pull request merged**